resources/definitions/os_discovery/eltex-mes24xx.yaml

```diff
@@ -62,7 +62,7 @@ modules:
                 -
                     oid: eltMesIssEnvFanTable
                     value: eltMesIssEnvFanSpeedLevel
-                    num_oid: '.1.3.6.1.4.1.35265.1.139.12.1.5.1.1.2.{{ $index }}'
+                    num_oid: '.1.3.6.1.4.1.35265.1.139.12.1.5.1.1.1.{{ $index }}'
```

Comment:
According to the MIB, `.2` is correct for FanSpeedLevel. If you change the num_oid to `.1` then you must also change the value to eltMesIssEnvFanSpeed. 

```

eltMesIssEnvFanSpeed OBJECT-TYPE
    SYNTAX     Integer32 (0..65535)
    MAX-ACCESS read-only
    STATUS     current
    DESCRIPTION
        "Numeric representation of the switch fan speed in RPM."
    ::= { eltMesIssEnvFanEntry 1 }

eltMesIssEnvFanSpeedLevel OBJECT-TYPE
    SYNTAX     Integer32 (0..3)
    MAX-ACCESS read-only
    STATUS     current
    DESCRIPTION
        "Indicates the switch fan speed level (0 - lowest, 3 - highest)."
    ::= { eltMesIssEnvFanEntry 2 }
```